ckanext/dgu/bin/fix_mandate.py

```diff
@@ -0,0 +1,62 @@
+'''
+Fix the secondary theme to be a JSON list (from list of Python unicode strings)
+
+e.g.
+
+[u'Crime'] -> ['Crime']
+'''
+
+import sys
+import common
+import json
+from optparse import OptionParser
+from ckan import model
+
+from running_stats import StatsList
+
+stats = StatsList()
+
+class FixMandate(object):
+    @classmethod
+    def command(cls, config_ini, write):
+        common.load_config(config_ini)
+        common.register_translator()
+
+        rev = model.repo.new_revision()
+        rev.author = 'fix_mandate.py'
+
+        for package in model.Session.query(model.Package):
+            if 'mandate' in package.extras:
+                stats.add('Fixing', package.name)
+
+                mandate = package.extras.get('mandate')
+
+	        package.extras['mandate'] = json.dumps([mandate, "", ""])
```

Comment:
I'd rather the blanks were added when displaying the form, rather than exist in the data. That's what we do for resources. Could you remove any blanks in the form on form submission by using a validator? I think resources do it that way.